frontend/src/main/components/PersonalSchedules/PersonalScheduleForm.js

```diff
@@ -15,25 +15,28 @@ function PersonalScheduleForm({
   const { data: systemInfo } = useSystemInfo();
   // Stryker disable OptionalChaining
   const startQtr = systemInfo?.startQtrYYYYQ || "20211";
-  const endQtr = systemInfo?.endQtrYYYYQ || "20214";
+  const endQtr = systemInfo?.endQtrYYYYQ || "20244";
```

Comment:
Why are we overwriting the default endQtr here? This should be defined in your .env and your dokku config variables.

Reply:
Ya, I was playing around with that because I thought it was messing with my code. I changed it back now.